Header header

uint8 evt_visual_track
uint8 evt_sound_track
uint8 evt_servo_stuck


uint8 VTRACK_DONE = 1
uint8 VTRACK_LOST = 2
